mxenabled/connect-widget
# used post message context instead of action

#### Issue 
https://mxcom.atlassian.net/browse/CT-932

#### Changes

- use context instead of post message action

#### Testing Instructions

- Connect to MXConnect and make sure that post messages are being sent with the correct metadata
** test using this MR: https://gitlab.mx.com/mx/connect-widget/-/merge_requests/217
